app/services/booking_responder/accept.rb

```diff
@@ -5,8 +5,28 @@ def process_request(message = nil)
         visit.rejection = nil
         visit.accept!
 
-        BookingResponse.new(success: true)
+        if visit.book_to_nomis_opt_out?
```

Comment:
As discussed in person, the problem is not the POST failing, it is succeeding and then the pvb having an error. The idea with this approach is to delay as much as possible the API call to avoid  other errors leaving PVB and Nomis inconsistent.